lib/index.js

```diff
@@ -799,11 +813,14 @@ Dataviz.version = pkg.version;
 function buildDomWrapper(el, options) {
   let html = '';
   let chart100percentWide = '';
+  let rangeChart;
```

Comment:
let rangeChart = '';
otherwise it'll render "undefined" in the html below (if it's not a range chart)